### PR TITLE
Tbcm 44 table legend

### PR DESCRIPTION
**Pull request by PeterPavlatos**
Created Activity Gap legend

- should toggle open / close
- open button
- list of icons with descriptions
- close button

image

### CODE REVIEW

File: apps/api/src/database/database.module.ts [hunk 1]
```diff
@@ -47,8 +47,8 @@ const environmentSpecificConfig = getEnvironmentSpecificConfig(nodeEnv);
 const appOrmConfig: PostgresConnectionOptions = {
   ...config,
   ...environmentSpecificConfig,
-  migrationsRun: true,
-  synchronize: false,
+  migrationsRun: false,
```
Comment (karthikktamilmani-freshworks) [inline apps/api/src/database/database.module.ts:50]: This can be removed

File: apps/web/src/common/constants.ts [hunk 2]
```diff
@@ -100,3 +100,36 @@ export const tooltipIcons: { [key in TooltipIconTypes]: TooltipIconProps } = {
     style: 'gray-icon',
   },
 };
+
+export const activityGapLegend = [
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/common/constants.ts:104]: I think we can re-use Leon's constants which had the same content

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 3]
```diff
@@ -21,6 +24,8 @@ export const RightSideBarActivites: React.FC = () => {
       setItems(newCareActivity);
       //Set checked ids to previously selected ids when care activity changes
       values.careActivities = _.filter(values.careActivityBundle[values.careActivityID]);
+      //Empty array when careActivityID changes
+      values.checked = [];
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/RightSideBarActivities.tsx:28]: Have already added some comments on the Tags PR regarding this

File: apps/web/src/components/RightSideBarActivities.tsx [hunk 4]
```diff
@@ -82,8 +84,14 @@ export const RightSideBarActivites: React.FC = () => {
               filteredData &&
               filteredData.map((item: any) => {
                 return (
-                  <div key={item.id}>
-                    <Checkbox name='careActivities' value={item.id} label={item.name} />
+                  <div key={item.id} className='flex flex-1 items-center'>
+                    <div className='flex-initial w-5/6'>
+                      <Checkbox name='checked' value={item.id} label={item.name} />
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/RightSideBarActivities.tsx:89]: Refer tags PR comments

File: apps/web/src/common/constants.ts (hunk 2 above)
Comment (PeterPavlatos) [reply apps/web/src/common/constants.ts]: Re used Leons  object.

File: apps/web/src/components/RightSideBarActivities.tsx (hunk 3 above)
Comment (PeterPavlatos) [reply apps/web/src/components/RightSideBarActivities.tsx]: Removed and updated to develop version.

File: apps/web/src/components/RightSideBarActivities.tsx (hunk 4 above)
Comment (PeterPavlatos) [reply apps/web/src/components/RightSideBarActivities.tsx]: Removed and updated to develop version.

File: apps/api/src/database/database.module.ts (hunk 1 above)
Comment (karthikktamilmani-freshworks) [reply apps/api/src/database/database.module.ts:50]: This shouldn't be pushed to dev and test envs.

**Review by karthikktamilmani-freshworks — COMMENTED** [review]
Looks great!

**Review by karthikktamilmani-freshworks — APPROVED** [review]

**Pull request merged**